.github/workflows/build.yaml

```diff
@@ -71,6 +71,12 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
 
+      - name: gofmt
```

Comment:
I added fmt step only in linux.
Because Jerome1337/gofmt-action@v1.0.5 is available in linux only.